pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/elasticExport/DeletedDataExportTest.java

```diff
@@ -0,0 +1,98 @@
+package org.broadinstitute.dsm.elasticExport;
+
+import java.util.List;
+import java.util.Map;
+
+import org.broadinstitute.dsm.DbAndElasticBaseTest;
+import org.broadinstitute.dsm.db.OncHistoryDetail;
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.export.ExportToES;
+import org.broadinstitute.dsm.model.elastic.search.ElasticSearch;
+import org.broadinstitute.dsm.pubsub.DSMtasksSubscription;
+import org.broadinstitute.dsm.statics.ESObjectConstants;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.broadinstitute.dsm.util.ElasticTestUtil;
+import org.broadinstitute.dsm.util.OncHistoryTestUtil;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DeletedDataExportTest extends DbAndElasticBaseTest {
+    private static final String instanceName = "delete_instance";
+    private static final DDPInstanceDao ddpInstanceDao = new DDPInstanceDao();
+    private static final String groupName = "delete_group";
+    private static String esIndex;
+    private static DDPInstanceDto ddpInstanceDto;
+    private static ParticipantDto testParticipant = null;
+    static OncHistoryTestUtil oncHistoryTestUtil;
+    static String userEmail = "deleteTestUser1@unittest.dev";
+    String guid = "DELETE_PARTICIPANT";
+    String guid1 = "DELETE1_PARTICIPANT";
+    String guid2 = "DELETE2_PARTICIPANT";
+    String guid3 = "DELETE3_PARTICIPANT";
+
+    @BeforeClass
+    public static void doFirst() {
+        esIndex = ElasticTestUtil.createIndex(instanceName, "elastic/lmsMappings.json", null);
```

Comment:
I understand your concern but LMS and OS2 are the best options for these features since their biggest usecase would be in the PECGS studies. I can get a mapping from OS index and try a test with that as well

Reply:
Ah that makes perfect sense--we have just the right index for the testing, in that case :)